core/templates/dev/head/pages/base.html

```diff
@@ -84,6 +84,7 @@
           '{{can_create_collections|js_string}}'),
         csrf_token: JSON.parse('{{csrf_token|js_string}}'),
         DEV_MODE: JSON.parse('{{DEV_MODE|js_string}}'),
+        errorCode: JSON.parse('{{code}}' | '0'),
```

Comment:
It's entirely possible that, one day, the backend will send a variable called 'code' and there's going to be a collision here.

Let's modify the backend (see base.py in core/controllers) to always send status_code (just like it sends csrf_token). Then read it here exactly the same way as you would for csrf_token, and use GLOBALS.status_code where needed.